packages/augur-ui/src/modules/reporting/common.tsx

```diff
@@ -713,9 +706,40 @@ export class ReportingBondsView extends Component<
             />
           </div>
         )}
+        {migrateRep &&
+        createBigNumber(inputtedReportingStake.inputStakeValue).lt(userAttoRep) && (
+          <DismissableNotice
+            show={true}
+            description=""
+            title="Are you sure you only want to migrate a portion of your REP to this universe?
+            If not, go back and select the ‘Max’ button to migrate your full REP amount."
+            buttonType={DISMISSABLE_NOTICE_BUTTON_TYPES.NONE}
+          />
+        )}
+        {migrateRep && (
+          <div
+            className={Styles.ReportingBondsViewCheckbox}
+            role="button"
+            tabIndex={0}
+            onClick={(e: React.SyntheticEvent) => {
+              e.preventDefault();
+              this.checkCheckbox(!readAndAgreedCheckbox);
+            }}
+          >
+            <label htmlFor="migrate-rep-confirmation">
+              <Checkbox
+                id="migrate-rep-confirmation"
+                isChecked={readAndAgreedCheckbox}
+                onClick={() => this.checkCheckbox(!readAndAgreedCheckbox)}
+                disabled={false}
+              />
+              I have carefully read all the information and fully acknowledge the consequences of migrating my REP to an unsuccessful universe
+            </label>
+          </div>
+        )}
         <PrimaryButton
-          text="Confirm"
-          disabled={disabled}
+          text={migrateRep ? "Confirm and Migrate REP" : "Confirm"}
+          disabled={migrateRep ? !readAndAgreedCheckbox : disabled}
```

Comment:
need to make sure there is no errors before enabling the button. 

issue here is, an error and button will still be active if they click the box.